client.py

```diff
@@ -1,23 +1,31 @@
 """This module facilitates communication between the show computer and the motion computer."""
 
 import socket
+import struct
 
+saved_num = 1
 class Client:
-    """This class contains all of the neccesary functions to initiate communication with the motion computer, send data to the motion computer, and close the socket conenction to the motion computer."""
+    """This class contains all of the neccesary functions to initiate
+    communication with the motion computer, send and receive data with the motion computer, and close the socket conenction to the motion computer."""
     def __init__(self, ip: str, port: int):
         """Initializes objects needed for network connection."""
         self.ip = ip
         """The IP of the motion computer."""
         self.port = port
         """The port that the motion computer will communicate on."""
         self.client_socket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-        """The socket object used for communication between the show computer and the motion computer."""
+        """The client socket for sending and receiving"""
+        # Bind the socket
+        print(ip)
+        self.client_socket.bind(('10.10.101.91', 4001)) #4002 when testing locally
```

Comment:
Should this IP be hardcoded? Is this supposed to point to localhost or the show computer?